src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -509,16 +510,22 @@ private function getShell()
         return self::$shell;
     }
 
-    private function isTty(): bool
+    private function isTty(OutputInterface $output): bool
     {
-        $inputStream = !$this->inputStream && \defined('STDIN') ? STDIN : $this->inputStream;
+        if (!$output instanceof StreamOutput) {
+            return true;
+        }
+
+        if (\DIRECTORY_SEPARATOR === '\\' && \function_exists('sapi_windows_vt100_support')) {
+            return @sapi_windows_vt100_support($output->getSteam());
+        }
 
         if (\function_exists('stream_isatty')) {
-            return stream_isatty($inputStream);
+            return stream_isatty($output->getStream());
         }
 
         if (!\function_exists('posix_isatty')) {
```

Comment:
This one looks to be huge oversight, I accidentally negated this.

Reply:
fixed